vue-components/src/components/MenuItem.ts

```diff
@@ -1,4 +1,5 @@
 export type MenuItem = {
 	label: string;
 	description: string;
+	tag?: string; // for internal use to signal disallowed datatypes
```

Comment:
The comment here is misleading and confusing. The tag is not for disallowed datatypes, it's for datatypes with limited support and more importantly, that's for QueryBuilder and I expect reader of this component to understand the comment without even knowing what QB is.

Reply:
you are completely right.

Reply:
I changed it